horizon/main.go

```diff
@@ -0,0 +1,113 @@
+package horizon
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/stellar/go-stellar-base/horizon/structs"
+)
+
+const TIMEOUT = 30 * time.Second
+
+// DefaultTestNetClient is a default client to connect to test network
+var DefaultTestNetClient = &Client{URL: "https://horizon-testnet.stellar.org"}
+// DefaultPublicNetClient is a default client to connect to public network
+var DefaultPublicNetClient = &Client{URL: "https://horizon.stellar.org"}
+
+// HorizonError struct contains error and problem returned by Horizon
+type HorizonError struct {
+	Err     error
```

Comment:
`error` is an interface. I'm still returning `HorizonError`. I will explain in a thread with your comment regarding `HorizonError`.